### PR TITLE
✨ Allow skipping file upload, sameAs and fingerprint for ISCN registration

### PR DESCRIPTION
**Pull request by williamchong**
image
image
image

### CODE REVIEW

File: components/IscnUploadForm.vue [hunk 1]
```diff
@@ -96,7 +96,12 @@
             </div>
           </div>
           <!-- Publish btn -->
-          <div class="flex flex-row justify-end pt-[24px] text-medium-gray">
+          <div class="flex gap-[8px] justify-end pt-[24px] text-medium-gray">
+            <Button
+              @click="onSkipUpload"
+              :preset="submitBtnClasses"
+              >{{ $t('UploadForm.button.skip') }}
```
Comment (nwingt) [inline components/IscnUploadForm.vue:103]:
```suggestion
              :preset="submitBtnClasses"
              @click="onSkipUpload"
            >{{ $t('UploadForm.button.skip') }}
```

File: locales/en.json [hunk 2]
```diff
@@ -325,6 +327,7 @@
   "UploadForm.button.back": "Back",
   "UploadForm.button.confirm": "Confirm",
   "UploadForm.button.selectFile": "Select a file",
+  "UploadForm.button.skip": "Skip Uploading",
   "UploadForm.button": "Start Register",
```
Comment (nwingt) [inline locales/en.json:331]:
```suggestion
  "UploadForm.button.skip": "Skip Upload",
  "UploadForm.button": "Start Upload",
```

File: components/IscnRegisterForm.vue [hunk 3]
```diff
@@ -89,6 +94,30 @@
       >
         <ContentFingerprintLink v-if="ipfsHash"  :item="formattedIpfs" />
         <ContentFingerprintLink v-if="uploadArweaveId" :item="formattedArweave" />
+        <ContentFingerprintLink v-for="f, i in customContentFingerprints" :key="f + i" :item="f" />
+        <div
+          v-if="!fileData"
+          :class="[
+            'flex',
+            'mt-[12px]',
```
Comment (nwingt) [inline components/IscnRegisterForm.vue:102]:
```suggestion
            { 'mt-[12px]': customContentFingerprints.length },
```

File: components/IscnRegisterForm.vue [hunk 4]
```diff
@@ -30,6 +30,7 @@
           'text-medium-gray',
         ]"
       >
+        <template v-if="fileData">
         <Previewer :is-image="isImage" :file-data="fileData" />
```
Comment (nwingt) [inline components/IscnRegisterForm.vue:34]: Indent?

File: components/IscnRegisterForm.vue [hunk 5]
```diff
@@ -89,6 +94,30 @@
...
+            class="w-full"
+          />
+          <Button
+            type="button"
+            class="mb-[4px]"
+            size="mini"
+            preset="secondary"
+            content-class="py-[4px]"
+            @click="addContentFingerprint"
+          >
```
Comment (nwingt) [inline components/IscnRegisterForm.vue:117]:
```suggestion
          <Button
            preset="secondary"
            type="button"
            @click="addContentFingerprint"
          >
```

File: components/IscnRegisterForm.vue [hunk 6]
```diff
@@ -89,6 +94,30 @@
...
+          v-if="!fileData"
+          :class="[
+            'flex',
+            'mt-[12px]',
+            'gap-[8px]'
+          ]"
+        >
+          <TextField
+            v-model="contentFingerprintInput"
+            class="w-full"
```
Comment (nwingt) [inline components/IscnRegisterForm.vue:108]: Placeholder?

Comment (williamchong) [reply components/IscnRegisterForm.vue:108]: Actually no idea what would be a suitable placeholder...

**Review by nwingt — APPROVED** [review]

**Pull request merged**